privateGPT.py

```diff
@@ -31,6 +30,26 @@ def main():
         case _default:
             print(f"Model {model_type} not supported!")
             exit;
+    
+    vector_store_type = os.environ.get('VECTOR_STORE', 'weaviate')
```

Comment:
I'd add the new env var to the .env file so give it visibility and update Readme accordingly